### ISSUE TITLE
Fix the epoch_bus_factory start_time_ms when deploy

### ISSUE THREAD

**Issue by jolestar** [movescriptions/movescriptions#81]
If the start_time_ms is before now, use now timestamp.

**Comment by RandyPen** [comment]
```Rust
    public fun do_deploy(
        deploy_record: &mut DeployRecord,
        tick_tick_record: &mut TickRecordV2, 
        tick_name: Movescription,
        total_supply: u64,
        init_locked_sui: u64,
        start_time_ms: u64,
        epoch_count: u64, 
        clk: &Clock,
        ctx: &mut TxContext
    ) {
        assert_util::assert_tick_tick(&tick_name);
        assert!(epoch_count >= MIN_EPOCHS, ErrorInvalidEpoch);
        
        let now = clock::timestamp_ms(clk);
        if (start_time_ms < now) {
            start_time_ms = now;
        };
        let tick_record = tick_factory::do_deploy(deploy_record, tick_tick_record, tick_name, total_supply, true, WITNESS{}, clk, ctx);
        after_deploy(tick_record, total_supply, init_locked_sui, start_time_ms, epoch_count, ctx);
    }
```

**Comment by jolestar** [comment]
fixed in #83
